### DISCUSSION TITLE
Clear the Error Log

### DISCUSSION THREAD

**Discussion by itsgucci** [owncast/owncast#1178] [Q&A]
I have just finished successfully configuring Owncast, and I would like to empty the error log so that from henceforth it will only show errors that actually happen in production. Is there an easy way to clear the error log?

**Comment by gabek** [comment]
If you restart the service the log in the admin will be cleared and start fresh.

**Reply by itsgucci** [reply]
awesome! thanks for the quick reply. restarting did indeed clear the log.
